mage_integrations/mage_integrations/sources/pipedrive/__init__.py

```diff
@@ -0,0 +1,44 @@
+from singer import catalog as catalog_singer
```

Comment:
nit: Could you sort the import statements in alphabetical order?
```
from mage_integrations.sources.base import Source, main
from mage_integrations.sources.catalog import Catalog
from mage_integrations.sources.pipedrive.tap_pipedrive.tap import PipedriveTap
from mage_integrations.utils.dictionary import ignore_keys
from singer import catalog as catalog_singer
from typing import List
import json

Reply:
Fixed on commit 5032fbfa4c5701035425bc14e47fc2a6d4897da5

Reply:
oh, didn't notice the format, sorry =P. Fixing it again

Reply:
Resorted on commit 45adb97fe41d6f6a914a3d8296475ebec3eb1864